src/Components/Plots/FullPlotly.tsx

```diff
@@ -77,6 +77,14 @@ export default class FullPlotly extends React.Component<
     if (queryParamList.indexOf('') !== -1) {
       queryParamList.splice(queryParamList.indexOf(''), 1)
     }
+
+    var tmp = []
+    for (var i = 0; i < queryParamList.length; i++) {
+      if (tmp.indexOf(queryParamList[i]) == -1) {
+        tmp.push(queryParamList[i])
+      }
+    }
+    queryParamList = tmp
```

Comment:
```suggestion
    queryParamList = [...new Set(queryParamList]
```

Reply:
causes an error because of --downlevelIteration flag not set in the TS compiler. Not sure what the implication of this is so I am skipping this for now.

Reply:
We could try:
```suggestion
    queryParamList = queryParamList.filter((v,i,a) => a.indexOf(v) === i)
```